README.md

```diff
@@ -39,6 +39,10 @@ Add a list of npm packages with a `name` and (optional) `version` to be installe
       # Install the latest stable release of a package.
       - name: node-sass
 
+Set a path pointing to a package.json. This will install all of the packages globally.
+
+    nodejs_npm_global_packages: "/var/www/application/package.json"
```

Comment:
This variable needs to be: `nodejs_package_json_path: ""`

Also, please place the description of the variable below the variable itself.